src/components/Footer/FooterClosing/styled.js

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components';
+
+import DefaultLink from 'components/Link';
+import { footerText } from 'styles/colours';
+import { spacing } from 'styles/spacing';
+import breakpoints from 'styles/breakpoints';
+
+export const Container = styled.section`
+  color: ${footerText};
+  display: flex;
```

Comment:
Doing this in a lot of places. It might be worth putting together a flexbox-based grid system, or a css grid based one.